src/chain.h

```diff
@@ -232,7 +232,7 @@ class CBlockIndex
 
     uint256 GetBlockHash() const
```

Comment:
nit, could change to `const uint256&`? Or drop it?

Reply:
done!